op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -53,8 +75,8 @@ func TestExecFromSameAddressInALoop(gt *testing.T) {
 	// Wait to include the exec txs until we know it will be included at a higher timestamp than initMsgsTx, modulo reorgs.
 	// NOTE: this should be `<`, but the mempool filtering in op-geth currently uses the unsafe head's timestamp instead of
 	// the pending timestamp. See https://github.com/ethereum-optimism/op-geth/issues/603.
-	for sys.L2ChainB.UnsafeHeadRef().Time <= initMsgsTx.PlannedTx.IncludedBlock.Value().Time {
-		sys.L2ChainB.WaitForBlock()
+	for l2ELA.BlockRefByLabel(eth.Unsafe).Time <= initMsgsTx.PlannedTx.IncludedBlock.Value().Time {
+		l2ELA.WaitForBlock()
```

Comment:
Non-blocking: Would be nice if a WaitForTimestamp(eth.Unsafe, time) existed to simplify this line.

Reply:
Agreed, something like that could be nice. I would prefer `WaitForBlock` to be more configurable with `WithMinTimestamp`, `WithMinBlock`, etc.

since it's not relevant for this PR, I'll leave it for future.